lib/tortoise311/connection.ex

```diff
@@ -407,7 +407,7 @@ defmodule Tortoise311.Connection do
          {:ok, state} = init_connection(socket, state) do
       # we are connected; reset backoff state, etc
       state =
-        %State{state | backoff: Backoff.reset(state.backoff)}
```

Comment:
If I understand the code here, this is the case where a successful connection happens. If that's right, I'd expect it to reset the backoff timeout back to the shortest time so that if the connection breaks later on, it will reconnect right away.